src/status_im/i18n_test.cljs

```diff
@@ -421,6 +421,7 @@
     :error
     :error-unable-to-get-balance
     :error-unable-to-get-prices
+    :error-usd-value-is-not-availble
```

Comment:
do we have English translation for this key?

Reply:
no we don't. should I add it?

Reply:
yes please, @hesterbruikman could you help with the copy ?